Gemfile

```diff
@@ -7,7 +7,10 @@ gem 'xcpretty'
 
 # A reference to Maze Runner is only needed for running tests locally and if committed it must be
 # portable for CI, e.g. a specific release.  However, leaving it commented out would mean quicker CI.
-gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v5.6.0'
+#gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v5.6.0'
+
+# Use a development branch
+gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', branch: 'tms/intelligent-document-server'
 
 # Locally, you can run against Maze Runner branches and uncommitted changes:
-# gem 'bugsnag-maze-runner', path: '../maze-runner'
+#gem 'bugsnag-maze-runner', path: '../maze-runner'
```

Comment:
`tms/intelligent-document-server` has now been [merged](https://github.com/bugsnag/maze-runner/commit/2656fec8c2f523ad60b58dcce1fc33e8d9d87ca9) so we should switch to using v5.11.0